components/financial_assistance/app/views/financial_assistance/benefits/index.html.erb

```diff
@@ -34,7 +34,7 @@
     </div>
 
     <div class='benefit-kinds', id='enrolled-benefit-kinds'>
-      <%= render partial: 'financial_assistance/benefits/insurance_kinds', locals: { insurance_kinds: FinancialAssistance::Benefit.valid_insurance_kinds, kind: "is_enrolled", text: "What type of coverage is this person currently enrolled in? *"} %>
+      <%= render partial: 'financial_assistance/benefits/insurance_kinds', locals: { insurance_kinds: FinancialAssistance::Benefit.valid_insurance_kinds, kind: "is_enrolled", text1: "What type of ", text2: " is this person currently enrolled in? *"} %>
```

Comment:
The way the rendering was being handled in _insurance_kinds.html.erb, you couldn't use the tabindex on the glossary word. This was the most straightforward fix.